scripts/exportrestapi.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+if [ $# -eq 0 ]
+  then
+    echo "No arguments supplied."
+    echo "Expected Parameters:"
+    echo "  1. The Physical ID of the AWS::ApiGateway::RestApi, for example 'd82arw4bnk'"
+    exit 1
+fi
+today=$(date +"%Y-%m-%d")
+aws apigateway get-export --export-type swagger --parameters extensions='postman' --profile atat-sandbox-dev --stage-name prod --rest-api-id $1 ~/AtatWebApi-restapi-$1-postman-export-${today}.json
```

Comment:
Why Swagger and not OpenAPI v3?

Reply:
Let's not hardcode a profile

Reply:
I've made the improved version use environment var `AWS_PROFILE` with a sensible default.

As for the Swagger vs OpenAPI: I'm not sure.  Been using these settings for several sprints and it works with Postman.  Will look into it.

Reply:
`oas30` would be better here

> [Required] The type of export. Acceptable values are 'oas30' for OpenAPI 3.0.x and 'swagger' for Swagger/OpenAPI 2.0.

Reply:
Changed in 4dfa3a8a5635fa364e99e34a44e577ecf59cf1f3